lib/manager/fvm/extract.ts

```diff
@@ -0,0 +1,36 @@
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import { logger } from '../../logger';
+import type { PackageDependency, PackageFile } from '../types';
+
+interface FvmConfig {
+  flutterSdkVersion: string;
+}
+
+export function extractPackageFile(
+  content: string,
+  packageFile: string
+): PackageFile | null {
+  let fvmConfig: FvmConfig;
+  try {
+    fvmConfig = JSON.parse(content);
+  } catch (err) {
+    logger.debug({ packageFile }, 'Invalid FVM config');
+    return null;
+  }
+
+  if (!fvmConfig.flutterSdkVersion) {
+    logger.debug(
+      { contents: fvmConfig },
+      'FVM config does not have flutterSdkVersion specified'
+    );
+    return null;
+  }
+
+  const dep: PackageDependency = {
+    depName: 'flutter',
+    currentValue: fvmConfig.flutterSdkVersion,
```

Comment:
We should make sure it's a plain string

Reply:
Added an extra check for this

Reply:
I don't think it will ever be a numerical value with something like `2.10`. Flutter versioning has been with the format of `major.minor.patch` since version `1.0.0`.